templates/zerver/api/outgoing-webhooks.md

```diff
@@ -103,15 +103,38 @@ A correctly implemented endpoint will do the following:
 * For a failed request, the endpoint should return data on the
   error.
 
-### Example payload
+### Example request payloads
+
+This is an example of the JSON payload that the Zulip server will `POST`
+to your server:
 
 {generate_code_example|zulip-outgoing-webhook-payload|fixture}
 
+### Example response payloads
+
+Here's an example of the JSON your server should respond with if
+you would not like to send a response message:
+
+```
+{
+    "response_not_required": true
+}
+```
+
+Here's an example of the JSON your server should respond with if
+you would like to send a response message:
+
+```
+{
+    "response_string": "Hey, we just received something from Zulip!"
```

Comment:
I'd like to move us to "content" here in the docs.  We just merged a change of mine that makes "content" the new key.  (We still support "response_string", but that wording doesn't match up with our API and bot frameworks.